coffeecard/CoffeeCard.WebApi/Controllers/v2/AccountController.cs

```diff
@@ -136,6 +135,25 @@ public async Task<ActionResult<EmailExistsResponse>> EmailExists([FromBody] Emai
             });
         }
 
+        /// <summary>
+        /// Updates the user group of a user
+        /// </summary>
+        /// <param name="id"> id of the user whose userGroup will be updated </param>
+        /// <param name="userGroup"> UserGroup object that will update the current userGroup attribute </param>
+        /// <returns> no content result </returns>
+        /// <response code="204"> The update was processed </response>
+        /// <response code="401"> Invalid credentials </response>
+        [HttpPatch]
+        [ProducesResponseType(typeof(void), StatusCodes.Status204NoContent)]
+        [ProducesResponseType(typeof(void), StatusCodes.Status401Unauthorized)]
+        [Route("{id:int}/user-group")]
+        public async Task<ActionResult> UpdateAccountUserGroup(int id, UserGroup userGroup)
```

Comment:
Refactor to take a object as requestbody instead of the the UserGroup property. Reason being having a requestbody object allows for easier refactoring with out introducing breaking changes for clients